WebHostLib/static/assets/player-settings.js

```diff
@@ -304,15 +304,15 @@ const toggleRandomize = (event, inputElement, optionalSelectElement = null) => {
     if (optionalSelectElement) {
       optionalSelectElement.disabled = undefined;
     }
+    updateGameSetting(inputElement);
   } else {
     randomButton.classList.add('active');
     inputElement.disabled = true;
     if (optionalSelectElement) {
       optionalSelectElement.disabled = true;
     }
+    updateGameSetting(randomButton);
   }
-
```

Comment:
I prefer fewer lines of code where possible. The same can be accomplished by using a ternary operator.
```js
updateGameSetting(active ? inputElement : randomButton);
```
Because `active` is already stored as a `const`, it's very cheap to evaluate again.

Reply:
Ultimately either way is fine. This is a matter of preference.

Reply:
I agree, less code is better. Thanks for pointing it out, I didn't even think about it